worker/include/RTC/PipeConsumer.hpp

```diff
@@ -10,6 +10,13 @@ namespace RTC
 {
 	class PipeConsumer : public RTC::Consumer, public RTC::RtpStreamSend::Listener
 	{
+	private:
+		static void StorePacketInTargetLayerRetransmissionBuffer(
```

Comment:
I don't like that we are moving method to static methods just because clang-tidy complains.